similar/editor/med.cpp

```diff
@@ -891,7 +891,7 @@ static void close_editor()
 {
 	//	_MARK_("end of editor");//Nuked to compile -KRB
 	
-#ifndef __linux__
+#if !defined (__linux__) || !defined(__OpenBSD__)
```

Comment:
This seems wrong to me.  Linux is not OpenBSD, so `(!linux || !openbsd)` -> `!(linux && openbsd)` -> `!true` -> `false`.  What is your goal here?  Do you want OpenBSD to use the same path as Linux, a path which calls `clear_warn_func`?

Reply:
Yes, that was my goal. I guess I messed up the logic and this should be !linux && !openbsd

Reply:
Build failed without the change because of msgbox_warning which I think is Windows specific?? I fixed the logic locally, stay tuned for updated pull request.